src/python/dxpy/cli/parsers.py

```diff
@@ -313,8 +313,18 @@ def __call__(self, parser, namespace, values, option_string=None):
         print()
         print('Available instance types:')
         print()
-        print(format_table(InstanceTypesCompleter.instance_types.values(),
-                           column_names=list(InstanceTypesCompleter.instance_types.values())[0]._fields))
+        print(format_table(InstanceTypesCompleter.standard_instance_types.values(),
+                           column_names=list(InstanceTypesCompleter.standard_instance_types.values())[0]._fields))
+        print()
+        print('Available GPU instance types:')
```

Comment:
```suggestion
        print('GPU instance types:')
```

Reply:
Trying to make this header a sub-header of "Available instance types" in line 314